airbyte-webapp/src/packages/cloud/views/auth/SignupPage/components/SpecialBlock.module.scss

```diff
@@ -0,0 +1,21 @@
+@use "../../../../../../scss/colors";
+
+.text {
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 17px;
+  margin: 0;
+
+  &:before {
+    content: "\1F389";
```

Comment:
Is there a specific reason, we're not adding this as part of the message simply (which would then also allow adjusting it via LaunchDarkly experimentation)?

Reply:
I had a similar question before and it's based on how the design is laid out with very specific quote placement that is difficult to reproduce without it.

<img width="114" alt="image" src="https://user-images.githubusercontent.com/168664/178350634-f582a6b1-418f-4600-9cc8-8228f0eef78b.png">

Alternatively, we could simplify the quote style.

Reply:
Maybe I've checked that wrong. I thought this is the party cone in front of the subtitle, given the Unicode?

Reply:
I was looking at the wrong section. This may be an emoji. Should we the emoji directly to the string?

Reply:
I think it would make more sense. So we'd have it available for experimentation (and potentially for later translation it could make sense using different emojis - or none - in a specific language)

Reply:
Updated 🎉